static/openapi/prerendering-spec.json

```diff
@@ -4335,525 +4691,631 @@
         }
       }
     },
-    "/api/prerender/v1/origins/{origin}/prefetch/urls": {
+    "/api/prerender/v1/history/render": {
```

Comment:
This should be under /api/prerender/v1/metrics.

This is again going into details about his APIs, but none of the others do this.  A lot of the details are already defined in the swagger/openapi definition below.

Reply:
Yeah, I wonder if we should be too detailed in the endpoint description because lots of information here will be in the description of `attributes` / `headers` / `query params` / `path params`.

Reply:
Lets come back and review all the APIs.  Do that in a different PR.  So leave this one for now.